contracts/test/vault/oneinch-swapper.js

```diff
@@ -134,10 +134,8 @@ describe("1Inch Swapper", () => {
 
       const fromAmount = utils.parseEther("20");
 
-      for (const fromAsset of [weth, reth, stETH, frxETH]) {
-        for (const toAsset of [weth, reth, stETH, frxETH]) {
-          if (fromAsset.address === toAsset.address) continue;
-
+      for (const fromAsset of [weth]) {
```

Comment:
should we leave this in? Since we still might want to swap from LSTs to WETH

Reply:
These tests involve minting with LSTs as part of the fixture. This can get really flaky and unstable as we swap out the LSTs